### PR TITLE
NATIVE-1415-passenger-object-information

### PR DESCRIPTION
**Pull request by firminofranciele007**

### CODE REVIEW

File: docs/android/inpath/propertyDescriptions.md [hunk 1]
```diff
@@ -119,6 +119,41 @@ A string value that represents the Order ID for a Flight PNR or Booking Referenc
...
 
+The CTSdkPassenger object has the following optional parameters:
+
+| Parameter          | Description                                  | Usage                                                                                                                                       | Type      | 
+|:-------------------|:---------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| firstName          | Customer's name                              | First name displayed on the driver details' screen                                                                                          | String    |
+| lastName           | Customer's last name                         | Last name displayed on the driver details' screen                                                                                           | String    |
+| email              | Customer's email                             | Email displayed on the driver details' screen                                                                                               | String    |
```
Comment (MarkMan2407) [inline docs/android/inpath/propertyDescriptions.md:128]: I don't think driver details' should have the apostrophe at the end in any of these entries

File: docs/android/inpath/propertyDescriptions.md [hunk 2]
```diff
@@ -119,6 +119,41 @@ A string value that represents the Order ID for a Flight PNR or Booking Referenc
...
+        .phoneNumber("08666666666")
+        .address("Pearse Stree 738")
+        .city("Cork")
+        .postCode("W99NH99")
+        .country("IE")
+        .flightNumber("IE 123")
+        .age("29")
+        .membershipId("123456")
+        .build()
+```
```
Comment (MarkMan2407) [inline docs/android/inpath/propertyDescriptions.md:155]: Can you add an extra line here to show how the CTSdkPassenger object is used, like we did with the CTFlightDetails object

File: docs/ios/inpath/propertyDescriptions.md [hunk 3]
```diff
@@ -62,4 +62,47 @@ The `countryCode` property refers to the country of residency, and this is used
...
+                            city: "Dublin",
+                            postcode: "Dublin 14",
+                            countryCode: "IE",
+                            age: 25,
+                            email: "ryan.oconnor@cartrawler.com",
+                            phone: "0838880000",
+                            phoneCountryPrefix: "353",
+                            loyaltyProgramNumber: "1234",
+                            isPrimaryDriver: true)
+context.passengers = [passenger]
```
Comment (MarkMan2407) [inline docs/ios/inpath/propertyDescriptions.md:104]: @GHarte 'context' has no meaning on this page. Might be worth showing the creation of a context object here (eventhough it's mentioned on Implementation Steps). Same for the other section.

**Review by MarkMan2407 — CHANGES_REQUESTED** [review]

File: docs/android/inpath/propertyDescriptions.md (hunk 1 above)
Comment (firminofranciele007) [reply docs/android/inpath/propertyDescriptions.md]: done

File: docs/android/inpath/propertyDescriptions.md (hunk 2 above)
Comment (firminofranciele007) [reply docs/android/inpath/propertyDescriptions.md:159]: done

File: docs/ios/inpath/propertyDescriptions.md (hunk 3 above)
Comment (GHarte) [reply docs/ios/inpath/propertyDescriptions.md:109]: done

**Review by MarkMan2407 — APPROVED** [review]

**Review by MarkMan2407 — APPROVED** [review]

**Pull request merged**